dev/io.openliberty.org.jboss.resteasy.mprestclient.4.0/src/io/openliberty/microprofile/rest/client40/internal/OsgiServices.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.rest.client40.internal;
+
+import java.util.Optional;
+import java.util.concurrent.ExecutorService;
+
+import jakarta.enterprise.concurrent.ManagedExecutorService;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+
+/**
+ * Provides access to OSGi services to non-osgi components
+ */
+@Component(configurationPolicy = ConfigurationPolicy.IGNORE)
+public class OsgiServices {
```

Comment:
Will need to determine if this is needed any longer since it isn't used by any code.  More on this in a later comment.

Reply:
Still needed.  See #28960